modules/changelog.lua

```diff
@@ -0,0 +1,107 @@
+local Event = require 'utils.event'
```

Comment:
This module does not make much sense, we have map info to list these things.

Reply:
I wanted to separate between the changes that were made to a particular map and the information about a map. I can certainly modify it so that I can put a changelog at the end of the map. It seemed nice to let people playing know what had changed since the last version.

Reply:
Valid reason, ty!